src/book/02-projects/01-nubots/11-flashing.mdx

```diff
@@ -0,0 +1,169 @@
+---
+section: Projects
+chapter: NUbots
+title: Flashing a Robot
+description: How to installing a new operating system on the robot
+slug: /projects/nubots/flashing
+---
+
+## Flashing a Robot
+
+To install Arch Linux (our OS of choice) on to a robot perform the following instructions
+
+1. Download the latest [Arch Linux LiveUSB](https://www.archlinux.org/download/) image and burn it on to a [USB thumb drive](https://wiki.archlinux.org/index.php/USB_flash_installation_media#In_GNU/Linux)
+
+   - Find the USB drive in your system. This will vary from one system to another and choosing the wrong drive will cause irreversible data loss.
+     - Use `sudo fdisk -l`, or the `Disks` GUI program to find your USB drive. It will probably be `/dev/sdX`, where `X` is a letter of the alphabet.
+   - Burn the Arch Linux ISO you just downloaded to the USB drive (in this example we are using `/dev/sdb`)
+   - ```sh
+     sudo dd bs=4M if=~/Downloads/archlinux-2019.11.01-x86_64.iso of=/dev/sdb status=progress oflag=sync
+     ```
+
+1. If you are using a brand new NUC, Boot into the BIOS on the NUC
+
+   - Turn the NUC on and hold down the F2 button
+   - Intel Visual BIOS should appear
+   - Navigate to "Advanced" menu
+   - Navigate to "Power" then set "After Power Failure" to "Power On"
+     ![BIOS Power Menu](./images/flashing/bios_power.jpg 'BIOS Power Menu')
+   - Navigate to "Boot" and then "Boot Configuration"
+     - Ensure "Boot USB Devices First" is enabled in "UEFI Boot"
+     - Ensure "USB" is enabled in "Boot Devices"
+       ![BIOS Boot Configuration Menu](./images/flashing/bios_boot_configuration.jpg 'BIOS Boot Configuration Menu')
+   - Navigate to "Secure Boot" and disable "Secure Boot"
+     ![BIOS Secure Boot Menu](./images/flashing/bios_secure_boot.jpg 'BIOS Secure Boot Menu')
+   - Press F10 to save settings and exit and ensure you USB drive is plugged in
+
+1. If you are not using a brand new NUC, ensure your USB drive is plugged in and turn the NUC on
```

Comment:
Numbering:

```suggestion
2. If you are not using a brand new NUC, ensure your USB drive is plugged in and turn the NUC on
```

Same with 3, 4 and 5 below, as well as in **Manually setting up a WiFi connection on the robot**.

Or is the numbering messed up when you format the file?